tests/requirements-relaxed.txt

```diff
@@ -69,18 +71,20 @@ multidict==6.0.5
     # via
     #   aiohttp
     #   yarl
-packaging==23.2
+packaging==24.0
     # via
     #   antsibull-core
     #   antsibull-docs
     #   build
     #   sphinx
 perky==0.9.2
     # via antsibull-core
-pydantic==1.10.14
+pydantic==2.6.4
```

Comment:
antsibull-core v3 now supports pydantic v2, so we see this major bump here. The model schemas that parse/validate the ansible-docs output still use the old pydantic v1 code[^1], so those _should_ not introduce any regressions.

[^1]: It's complicated :sigh:. See https://github.com/ansible-community/antsibull-docs/pull/224 if your curious.